### ISSUE TITLE
Introspection doesn't work when using `prisma init` with an existing PostgreSQL DB

### ISSUE THREAD

**Issue by nikolasburk** [prisma/prisma#3866]
**Describe the bug**

Introspection doesn't work when using `prisma init` with an existing PostgreSQL DB

**To Reproduce**

I'm using a Postgres DB on Heroku looking like this:

```sql
-- Sequence
CREATE SEQUENCE IF NOT EXISTS "public"."post_id_seq";

-- Table Definition
CREATE TABLE "public"."post" (
    "id" int4 NOT NULL DEFAULT nextval('post_id_seq'::regclass),
    "title" varchar NOT NULL,
    "author" int4 NOT NULL,
    CONSTRAINT "post_author_fkey" FOREIGN KEY ("author") REFERENCES "public"."user"("id") ON DELETE CASCADE ON UPDATE CASCADE,
    PRIMARY KEY ("id")
);
```

```sql
-- Sequence
CREATE SEQUENCE IF NOT EXISTS "public"."user_id_seq";

-- Table Definition
CREATE TABLE "public"."user" (
    "id" int4 NOT NULL DEFAULT nextval('user_id_seq'::regclass),
    "name" varchar NOT NULL,
    "email" varchar,
    PRIMARY KEY ("id")
);
```

Now I'm running `prisma init`:

![image](https://user-images.githubusercontent.com/4058327/51171296-d813d580-18b0-11e9-9974-1643d99ff0c5.png)

The generated datamodel looks like this:

```graphql
type User {
  id: ID! @unique
  name: String!
}
```

The generated Docker Compose file looks like this:

```yml
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.23
    restart: always
    ports:
    - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        # uncomment the next line and provide the env var PRISMA_MANAGEMENT_API_SECRET=my-secret to activate cluster security
        # managementApiSecret: my-secret
        databases:
          default:
            connector: postgres
            host: ec2-54-235-169-191.compute-1.amazonaws.com
            database: d60c67pi42ha6
            schema: public
            user: nwbmapwrrgqtcu
            password: f9c5c7921be8fc3ac1a8323bf6aa48fd6645852eebcee7dbaf9d86e4f063c8cc
            rawAccess: true
            port: '5432'
            migrations: true
```

**Expected behavior**

The generated datamodel file should look like this:

```graphql
type Post @pgTable(name: "post") {
  author: User @pgRelation(column: "author")
  id: Int! @id @unique
  title: String!
}

type User @pgTable(name: "user") {
  email: String
  id: Int! @id @unique
  name: String!
  posts: [Post!]!
}
```

The generated `docker-compose.yml` should:

- use the Prisma Docker image `1.24` (I'm on `prisma/1.24.0 (darwin-x64) node-v10.13.0`)
-  have `migrations` set to `false`
- `ssl` set to `true`

**Comment by nikolasburk** [comment]
Follow up, I've adjusted the Docker Compose file to look like this:

```yml
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.24
    restart: always
    ports:
    - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        # uncomment the next line and provide the env var PRISMA_MANAGEMENT_API_SECRET=my-secret to activate cluster security
        # managementApiSecret: my-secret
        databases:
          default:
            connector: postgres
            host: ec2-54-235-169-191.compute-1.amazonaws.com
            database: d60c67pi42ha6
            schema: public
            user: nwbmapwrrgqtcu
            password: f9c5c7921be8fc3ac1a8323bf6aa48fd6645852eebcee7dbaf9d86e4f063c8cc
            rawAccess: true
            port: '5432'
            migrations: true
            ssl: true
```

However, starting the Docker container with `docker-compose up -d` fails. This is the Docker logs:

```
Jan 15, 2019 9:54:41 AM org.postgresql.Driver connect
SEVERE: Connection error:
org.postgresql.util.PSQLException: FATAL: permission denied for database "postgres"
  Detail: User does not have CONNECT privilege.
        at org.postgresql.core.v3.QueryExecutorImpl.receiveErrorResponse(QueryExecutorImpl.java:2433)
        at org.postgresql.core.v3.QueryExecutorImpl.readStartupMessages(QueryExecutorImpl.java:2566)
        at org.postgresql.core.v3.QueryExecutorImpl.<init>(QueryExecutorImpl.java:131)
        at org.postgresql.core.v3.ConnectionFactoryImpl.openConnectionImpl(ConnectionFactoryImpl.java:210)
        at org.postgresql.core.ConnectionFactory.openConnection(ConnectionFactory.java:49)
        at org.postgresql.jdbc.PgConnection.<init>(PgConnection.java:195)
        at org.postgresql.Driver.makeConnection(Driver.java:452)
        at org.postgresql.Driver.connect(Driver.java:254)
        at slick.jdbc.DriverDataSource.getConnection(DriverDataSource.scala:101)
        at com.zaxxer.hikari.pool.PoolBase.newConnection(PoolBase.java:341)
        at com.zaxxer.hikari.pool.PoolBase.newPoolEntry(PoolBase.java:193)
        at com.zaxxer.hikari.pool.HikariPool.createPoolEntry(HikariPool.java:430)
        at com.zaxxer.hikari.pool.HikariPool.access$500(HikariPool.java:64)
        at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:570)
        at com.zaxxer.hikari.pool.HikariPool$PoolEntryCreator.call(HikariPool.java:563)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by pantharshit00** [comment]
> However, starting the Docker container with docker-compose up -d fails. This is the Docker logs:

I have seen some users reporting the same issue. It works with `migrations: false` but I think we should allow this case as well.

**Comment by timsuchanek** [comment]
This has been fixed in today's release of `1.25`.
